corehq/apps/case_search/utils.py

```diff
@@ -99,21 +99,25 @@ def _add_daterange_queries(self):
             self.criteria.pop(key)
 
     def _add_case_property_queries(self):
-        try:
-            fuzzies = self.config.fuzzy_properties.get(
-                domain=self.domain, case_type=self.case_type).properties
-        except FuzzyProperties.DoesNotExist:
-            fuzzies = []
+        fuzzies = []
```

Comment:
This is interesting: fuzzy properties are per case type, but users will be searching across case types. So if a case property is set as fuzzy for one case type but not another, and you search across both types, it'll be treated as fuzzy for both case types, right? I think that's probably fine, but definitely something to include in docs.

Reply:
Same comment for the regex patterns, though I don't think anyone is using those.

Reply:
Yes, that's right. I have added this as a caveat in the spec and the product is okay with this behaviour. It's a sensible design choice, since searching across different types over same properties with same user inputs would indicate that the properties have same lexicon in most cases.